pymc4/coroutine_model.py

```diff
@@ -15,6 +15,7 @@
 def model(genfn, *, name=_no_name_provided, keep_auxiliary=True, keep_return=True, method=False):
     """Flexibly wrap a generator function into a Model template."""
     if method:
+        # What is this block for?
```

Comment:
This one code block :)

Reply:
In case first argument is self

Reply:
And if you do it another way, the defined method is not recognised as function, it is recognised as an object and self is not passed

Reply:
And if you do it another way, the defined method is not recognised as function, it is recognised as an object and self is not passed

Reply:
And if you do it another way, the defined method is not recognised as function, it is recognised as an object and self is not passed

Reply:
Thanks